XamlStyler.Package/StylerPackage.cs

```diff
@@ -278,9 +278,10 @@ private static IEnumerable<string> GetConfigPathBetweenPaths(string path, string
                 ? path
                 : Path.GetDirectoryName(path);
 
-            while (configDirectory.StartsWith(root, StringComparison.InvariantCultureIgnoreCase))
+            while (configDirectory != null && configDirectory.StartsWith(root, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Can we collapse this null check?

`while(configDirectory?.StartsWith(root, StringComparison.InvariantCultureIgnoreCase) ?? false)`

Reply:
done!